### ISSUE TITLE
Test that legacy platform object's [[DefineOwnProperty]] handles Symbols properly

### ISSUE THREAD

**Issue by tobie** [w3c/web-platform-tests#6912]
See heycam/webidl#411.

**Comment by bzbarsky** [comment]
@tobie, @TimothyGu is either of you planning to work on this?

**Comment by TimothyGu** [comment]
Yes, a WIP is https://github.com/w3c/web-platform-tests/pull/6786. I started working on Set but got burned out after seeing so many inconsistencies and thing to test...
